Animations/Fade/Left.js

```diff
@@ -26,5 +28,5 @@ var FadeLeft = {
     'transform': 'translateX(-20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeLeft', FadeLeft);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeLeft', FadeLeft);
```

Comment:
This does not need to have `'FadeLeft'` passed into it since we have defined a `className` property in `FadeLeft`.